src/pages/homepage/index.js

```diff
@@ -0,0 +1,149 @@
+import React from "react";
+import { Images } from "../../constants/images";
+import "./style.scss";
+
+function Homepage() {
+  return (
+    <div className="container">
+      <div className="card">
+        <div className="card-style left">
+          <div className="left-row-1">
+            <img src={Images.moon} alt="moon" className="moon" />
+            <div>
+              <button className="fahrenheit">F</button>
+              <button className="celcius">C</button>
+            </div>
+          </div>
+          <div className="left-row-2">
+            <p className="temp">
+              27<span>&#8451;</span>
+            </p>
+            <p className="date">17th jun' 21</p>
+            <div className="date-time">
+              <p>Thursday</p>
+              <p>|</p>
+              <p>2:45 am</p>
```

Comment:
Why different paragraph tags? This can be done by span elements. That would reduce lines in CSS as well.